.github/workflows/validate-tag.yml

```diff
@@ -16,17 +16,18 @@ jobs:
 
       - name: Setup Node.js and Cache
         uses: ./.github/actions/nodejs
+        id: nodejs-workspace
 
-      - name: Get last version from package.json
+      - name: Get last tag
         id: info
-        run: |
-          echo "::set-output name=package_version::v$(node -p "require('./package.json').version")"
-          echo "::set-output name=tag::$(git describe HEAD --abbrev=0)"
+        run: echo "::set-output name=tag::$(git describe HEAD --abbrev=0)"
 
       - name: Show an error if the tag does not match
-        if: ${{ steps.info.outputs.package_version != steps.info.outputs.tag }}
+        if:
+          ${{ steps.info.outputs.package_version != steps.info.outputs.tag &&
```

Comment:
Is `steps.info.outputs.package_version` defined ?

Reply:
fixed